packages/fx-core/resource/package.nls.json

```diff
@@ -113,6 +113,22 @@
   "plugins.frontend.checkSystemTimeTip": "You may get expired credentials, check if your system time is correct.",
   "plugins.frontend.restoreEnvironmentTip": "If you manually updated configuration files (under directory .%s), recover them.",
   "plugins.baseErrorMessage": "%s Suggestions: %s",
+  "plugins.bot.questionHostTypeTrigger.title": "Choose triggers",
```

Comment:
Nit: Better to align wording - it seems multi-env used `Select environments`.

Reply:
The wordings are from the PM spec: https://microsoftapc.sharepoint.com/:p:/t/DevDivTeamsDevXProductTeam/EX10MctyIRVFsO6r30kEq94BN1YA8BxLn8mx4kReQy1SbA?e=ZISk6G&nav=eyJzSWQiOjI4NywiY0lkIjozNDE3Mjk1MDE1fQ